kernel/src/table_changes/log_replay.rs

```diff
@@ -274,15 +275,18 @@ struct PreparePhaseVisitor<'a> {
     has_cdc_action: &'a mut bool,
     add_paths: &'a mut HashSet<String>,
     remove_dvs: &'a mut HashMap<String, DvInfo>,
+    commit_timestamp: &'a mut i64,
 }
 impl PreparePhaseVisitor<'_> {
     fn schema() -> Arc<StructType> {
+        let ict_type = StructField::new("inCommitTimestamp", DataType::LONG, true);
         Arc::new(StructType::new(vec![
             Option::<Add>::get_struct_field(ADD_NAME),
             Option::<Remove>::get_struct_field(REMOVE_NAME),
             Option::<Cdc>::get_struct_field(CDC_NAME),
             Option::<Metadata>::get_struct_field(METADATA_NAME),
             Option::<Protocol>::get_struct_field(PROTOCOL_NAME),
+            StructField::new("commitInfo", StructType::new([ict_type]), true),
```

Comment:
though i wonder if we can do something similar to above like `Option<CommitInfo>::get_struct_field(COMMIT_INFO_NAME)` and get struct field `inCommitTimestamp` of that?

but for now at least can use `COMMIT_INFO_NAME`?
```suggestion
            StructField::new(COMMIT_INFO_NAME, StructType::new([ict_type]), true),
```

Reply:
>  i wonder if we can do something similar to above like Option<CommitInfo>::get_struct_field(COMMIT_INFO_NAME) and get struct field inCommitTimestamp of that?

We would get a `StructField` of type CommitInfo, which we'd have to 1) get datatype, 2) cast to a struct 3) get the ICT field. So I'll stick with your suggested change 👍